File: HomeTestAngularShofiul/src/app/models/Project.ts
import { ProjectView } from './ViewModel/ProjectView';
import { ContactView } from './ViewModel/ContactView';

export class Project {
    constructor(
        public id: any ='',
        public projectName: any = '',
        public phone: any = '',
        public address: any = '',
        public projectDetails: any = '',
        public contactList: Array<any> = new Array<any>(),
        public contacts: Array<ContactView> = new Array<ContactView>()
    ) {}
}

File: HomeTestShofiul/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HomeTestShofiul.Models
{
    public class Project
    {
        public int Id { get; set; }
        public string ProjectName { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string ProjectDetails { get; set; }
        public virtual ICollection<ProjectContact> Contacts { get; set; }
    }
}
